iceberg/iceberg-1-9-x/src/main/scala/com/nvidia/spark/rapids/iceberg/IcebergProviderImpl.scala

```diff
@@ -0,0 +1,418 @@
+/*
```

Comment:
Identical to the file in 1-6-x module, move to common directory?
Is this intentional?

Reply:
Yes, it's intential since currently we are loading `IcebergProviderImpl` per shim, and this serves an example when we add more iceberg version support. Such little duplication is acceptable.